README.md

```diff
@@ -11,31 +11,8 @@ EasyPost is a simple shipping API. You can sign up for an account at https://eas
 npm install --save @easypost/api
 ```
 
-**Note:** if you are using a version of Node less than 6.9, you will need to install and
```

Comment:
Do we want to keep this? What if the user is using the older version of our Node CL?

Reply:
hmm, yeah we probably just need to clarify this to say if you're using our library prior to x version. I'll fix this.